clr.defines.targets

```diff
@@ -37,6 +37,7 @@
         <CDefines Condition="'$(FeatureDbgipcTransportVM)' == 'true'">$(CDefines);FEATURE_DBGIPC_TRANSPORT_VM</CDefines>
         <CDefines Condition="'$(FeatureDbgPublish)' == 'true'">$(CDefines);FEATURE_DBG_PUBLISH</CDefines>
         <CDefines Condition="'$(FeatureEventTrace)' == 'true'">$(CDefines);FEATURE_EVENT_TRACE</CDefines>
+        <CDefines Condition="'$(FeatureXplatEventSource)' == 'true'">$(CDefines);FEATURE_EVENTSOURCE_XPLAT</CDefines>
```

Comment:
Actually no, FEATURE_EVENT_TRACE is independent of FEATURE_EVENTSOURCE_XPLAT, they only interact by FireEtwEventSource function which can be defined independently of FEATURE_EVENT_TRACE